.gitattributes

```diff
@@ -0,0 +1,42 @@
+data/testfiles/Starmon/starmon_mini/v000/0-5T3990.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-5T4001.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/5T3987.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-4T3995.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/1T4102.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/4T3995.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/4T3999.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/5T3989.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-4T3997.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-4T3999.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-6T3863.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/4T3998.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/4T4068.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/5T3994.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/5T4001.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-4T3996.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/3T4098.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/3T4099.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/5T3992.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/5T4000.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-4T3769.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-4T3998.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/6T3863.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-5T3991.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-5T3992.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/0-5T4000.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/1T4100.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/1T4101.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v000/4T3996.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_mini/v001/petejan_starmonmini_4047_2019.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_oddi/v000/3X1010_saz15_star_oddi.DAT filter=lfs diff=lfs merge=lfs -text
+data/testfiles/Starmon/starmon_oddi/v000/4X1010.DAT filter=lfs diff=lfs merge=lfs -text
```

Comment:
Is this going to work when you try to compile on Windows?

Reply:
If you mean the forward-slash "/", yes it works.

I just searched for the reverse (people on windows writing gitattributes with backlashes "\"), and if people are using git-lfs>2.2.0 there should be no issues. I'' give it a go anyway to check before adding more stuff.

there is no rush also.  want this PR to have all test files related to v2.5.42 release before merging.